samples-tests/src/test/kotlin/org/gradle/kotlin/dsl/samples/MavenPublishSampleTest.kt

```diff
@@ -0,0 +1,43 @@
+package org.gradle.kotlin.dsl.samples
+
+import org.junit.Assert.assertTrue
+import org.junit.Test
+import java.nio.file.Paths
+import java.util.jar.JarFile
+
+class MavenPublishSampleTest : AbstractSampleTest("maven-publish") {
+    @Test
+    fun `publish`() {
+        val projectName = "maven-publish"
+        val projectVersion = "1.0.0"
+
+        // when:
+        build("publish")
+
+        // then:
+        val repo = projectRoot.toPath().resolve("build").resolve("repo")
+        val mavenPath = repo.resolve(Paths.get("org", "gradle", "sample", projectName, projectVersion))
```

Comment:
:x: Should use `existing("$repo/org/gradle/sample/$projectName/$projectVersion")` instead